recipe/meta.yaml

```
@@ -23,19 +22,25 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
+    - cython
     - setuptools
     - pip
     - wheel
   run:
-    - python >=3.6
+    - python
     # for fontTools.sfnt and fontTools.woff2: to compress/uncompress
     # WOFF 1.0 and WOFF 2.0 webfonts.
     - brotli >=1.0.1
     # 5/17/2021: the unicodedata2 extension module doesn't work on PyPy.
     # Python 3.9 already has Unicode 13.0, so the backport is not needed.
-    - unicodedata2 >=13.0.0  # [py<39 and (python_impl != "pypy")]
-    - munkres
+    - unicodedata2 >=14.0.0 # [py<311]
```

```suggestion
    - unicodedata2 >=15.1.0 # [py<312]
```
https://github.com/fonttools/fonttools/blob/4.49.0/setup.py#L111